### PR TITLE
Add logging when manifest can't be loaded

### PR DESCRIPTION
**Pull request by RomainBoulay**
### Short description 📝

> Bumped into an issue myself when working on describing a project with Tuist. And couldn't spot the (obvious) error before a while.

### Solution 📦

> Added more logging

Thanks a lot to @pepibumur who did the most of our pairing!

### CODE REVIEW

File: Sources/TuistLoader/Loaders/ManifestLoader.swift [hunk 1]
```diff
@@ -163,6 +163,10 @@ public class ManifestLoader: ManifestLoading {
             let manifestPath = path.appending(component: fileName)
             if !FileHandler.shared.exists(manifestPath) { continue }
             let data = try loadManifestData(at: manifestPath)
+            if Environment.shared.isVerbose {
+                let string = String(data: data, encoding: .utf8)
+                logger.debug("Could not load manifest, represented by this JSON:\n\(string)")
```
Comment (pepibumur) [inline Sources/TuistLoader/Loaders/ManifestLoader.swift:168]: At this point it's not true that we could not load the manifest. The line to load the manifest, and that could potentially raise an error is right below this. I'd change the message to:

```
"Trying to load parse the manifest represented by the following JSON representation:\n\(string)"
```

**Review by pepibumur — APPROVED** [review]
I left a nitpick comment @RomainBoulay. Besides that I'd update the CHANGELOG and I think we are good to merge if CI is happy about the changes.

**Comment by andreacipriani** [comment]
@RomainBoulay big boyscout

File: Sources/TuistLoader/Loaders/ManifestLoader.swift [hunk 2]
```diff
@@ -163,6 +163,10 @@ public class ManifestLoader: ManifestLoading {
             let manifestPath = path.appending(component: fileName)
             if !FileHandler.shared.exists(manifestPath) { continue }
             let data = try loadManifestData(at: manifestPath)
+            if Environment.shared.isVerbose {
+                let string = String(data: data, encoding: .utf8)
+                logger.debug("Could not load manifest represented by the following JSON representation:\n\(string)")
```
Comment (pepibumur) [inline Sources/TuistLoader/Loaders/ManifestLoader.swift:168]: @RomainBoulay I meant that the message shouldn't be "could not load", at this point we don't know if the JSON is valid or not. We could say:

```
Trying to load the manifest represented by the following JSON representation:\n\(string)
```

Comment (RomainBoulay) [reply Sources/TuistLoader/Loaders/ManifestLoader.swift]: Ha yeah, sorry I overlooked your previous comment 👍

**Pull request merged**